dev/com.ibm.ws.microprofile.reactive.streams.operators_fat/fat/src/com/ibm/ws/microprofile/reactive/streams/test/suite/FATSuite.java

```diff
@@ -54,6 +67,6 @@ public class FATSuite {
      * @return a RepeatTests instance
      */
     public static RepeatTests repeat(String server, TestMode otherFeatureSetsTestMode, FeatureSet firstFeatureSet, FeatureSet... otherFeatureSets) {
-        return RepeatActions.repeat(server, otherFeatureSetsTestMode, ALL, firstFeatureSet, otherFeatureSets);
+        return MicroProfileActions.repeat(server, otherFeatureSetsTestMode, ALL, firstFeatureSet, otherFeatureSets);
```

Comment:
This change should be reverted.

We wanted to call `MicroProfileActions` to make use of the logic which would change the firstFeatureSet in the case that we're running in LITE mode and it's not supported by the current java version.

However, the logic in `MicroProfileActions.repeat` assumes that all feature sets being passed in belong to the list of microprofile feature sets which isn't true here.